pydomcfg/tests/test_zco.py

```diff
@@ -6,7 +6,8 @@
 import xarray as xr
 
 from pydomcfg.domzgr.zco import Zco
-from pydomcfg.tests.bathymetry import Bathymetry
+
+from .bathymetry import Bathymetry
```

Comment:
Same thing, but I think it makes more sense to load it locally.
This is a module that we only use for testing.

Everything else, makes sense to import from the installed package.